client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -36,6 +37,7 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 
 	componentDidMount() {
 		this.loadInitializeScript();
+		window.addEventListener( 'message', this.receiveMessage, false );
```

Comment:
It's being removed in the `destroy` function, which is called by `componentWillUnmount`. `destroy` is also called in `componentDidUpdate`, since if the VideoPress ID changes, the player needs to be re-initialized.